packages/sommelier.rb

```diff
@@ -3,92 +3,93 @@
 class Sommelier < Package
   description 'Sommelier works by redirecting X11 programs to the built-in ChromeOS Exo Wayland server.'
   homepage 'https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/vm_tools/sommelier/'
-  version '20210109'
-  license '' # Can't find license. Probably BSD-Google or BSD-3
+  version '20210109-1'
+  license 'BSD-Google'
```

Comment:
Nice, we needed a license for this package.

Reply:
License from https://chromium.googlesource.com/chromiumos/platform2/+/refs/heads/main/LICENSE

Which appears to be the same as 

https://developers.google.com/open-source/licenses/bsd